go/private/sdk.bzl

```diff
@@ -17,6 +17,34 @@ load("//go/private:nogo.bzl", "go_register_nogo")
 load("//go/private/skylib/lib:versions.bzl", "versions")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "patch")
 
+patch_attrs = {
+    "patches": attr.label_list(
+        default = [],
+        doc =
+            "A list of files that are to be applied as patches to go sdk. " +
+            "By default, it uses the Bazel-native patch implementation " +
+            "which doesn't support fuzz match and binary patch, but Bazel will fall back to use " +
+            "patch command line tool if `patch_tool` attribute is specified or there are " +
+            "arguments other than `-p` in `patch_args` attribute.",
+    ),
+    "remote_patches": attr.string_dict(
```

Comment:
I would vote for dropping the `remote_*` attributes until we see a clear use case for them.

Reply:
I see no need to separate out the `patch_attrs` since they're only used once.

I update the docs in https://github.com/bazelbuild/rules_go/pull/3716

Closing this PR